lib/source/pl/lib/std/pragmas.cpp

```diff
@@ -3,8 +3,24 @@
 #include <pl/core/validator.hpp>
 #include <pl/core/evaluator.hpp>
 
-namespace pl::lib::libstd {
+const long unsigned int LIMITMAX = std::numeric_limits<long unsigned int>::max() - 1;
```

Comment:
This should probably be `constexpr static auto MaxLimit = std::numeric_limits<u64>::max();`